libs/game/tilemap.ts

```diff
@@ -8,18 +8,61 @@ namespace tiles {
         }
     }
 
+    /**
+     * A tile in the tilemap
+     **/
+    //% blockNamespace=scene color="#401255" blockGap=8
+    export class Tile {
+        private _row: number;
+        private _col: number;
+        private _index: number;
```

Comment:
I'm thinking if it wouldn't be better to pass the parent tile map and use `getPixelAt()` in the `tileSet` getter. You could then keep a reference to a tile and see it changing. Not sure what we want...

Reply:
Yeah, I guess that would be better. I see this API being used more in a "snapshot" scenario but it would look broken if you kept a reference to the tile around

Reply:
I like that idea, I'll change it real quick if that sounds good?

Reply:
Sounds good!